routing/route/route_test.go

```diff
@@ -197,6 +197,7 @@ func TestPayloadSize(t *testing.T) {
 			PubKeyBytes:      testPubKeyBytes,
 			AmtToForward:     1200,
 			OutgoingTimeLock: 700000,
+			ChannelID:        51784534844,
```

Comment:
> because we are building the route in reverse order I think this can never happen in real life

yeah, bad test values - will update in next push (though it doesn't _really_ matter in the context of this test).

> I would have liked this commit to be part of the commit where the validation is added, but that's just my personal taste

Reasoning for this is to demonstrate that these test values are valid before the stricter validation and after. I like having the updates to the tests in separate commits so that we're changing one thing at a time (rather than updating it with the validation, and then perhaps hitting a different code path by mistake).

> otherwise pretty hard to reason about this addition

Is there something that I could add to the commit message to clarify this? Did try to explain the reasoning in there.